modules/node/cms_node_api.e

```diff
@@ -327,6 +327,13 @@ feature -- Access: Node
 			end
 		end
 
+
+	available_parents_for_node (a_node: CMS_NODE): LIST [CMS_NODE]
+			-- Given the node `a_node', return the list of possible parent nodes
```

Comment:
This does not follow the Eiffel comment style.
But there is a comment, so I should not complain ;)